examples/Cpp/PrimitiveRestart/Example.140core.frag

```diff
@@ -0,0 +1,14 @@
+// GLSL shader version 1.40 (for OpenGL 3.2 Core Profile)
```

Comment:
Typo: GLSL 140 was introduced in OpenGL 3.1 Core Profile.

Reply:
The original mismatch is in the HelloTriangle sample. I just grabbed those shaders and bumped both the OpenGL and GLSL versions.

Reply:
Thanks, I'll fix those files myself then.